content/sensu-go/6.8/release-notes.md

```diff
@@ -110,6 +111,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.8.2 release notes
+
+**October 6, 2022** &mdash; The latest release of Sensu Go, version 6.8.2, is now available for download.
+
+Sensu Go 6.8.2
```

Comment:
Need to add the synopsis here; you may have already requested it

Reply:
Oh shoot